### PR TITLE
Combine some dep updates

### PR DESCRIPTION
**Pull request by doughsay**
combine all passing dependabot PRs

### CODE REVIEW

File: mix.lock [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
-  "idna": {:hex, :idna, "6.0.1", "1d038fb2e7668ce41fbf681d2c45902e52b3cb9e9c77b55334353b222c2ee50c", [:rebar3], [{:unicode_util_compat, "0.5.0", [hex: :unicode_util_compat, repo: "hexpm", optional: false]}], "hexpm", "a02c8a1c4fd601215bb0b0324c8a6986749f807ce35f25449ec9e69758708122"},
+  "hackney": {:hex, :hackney, "1.17.4", "99da4674592504d3fb0cfef0db84c3ba02b4508bae2dff8c0108baa0d6e0977c", [:rebar3], [{:certifi, "~>2.6.1", [hex: :certifi, repo: "hexpm", optional: false]}, {:idna, "~>6.1.0", [hex: :idna, repo: "hexpm", optional: false]}, {:metrics, "~>1.0.0", [hex: :metrics, repo: "hexpm", optional: false]}, {:mimerl, "~>1.1", [hex: :mimerl, repo: "hexpm", optional: false]}, {:parse_trans, "3.3.1", [hex: :parse_trans, repo: "hexpm", optional: false]}, {:ssl_verify_fun, "~>1.1.0", [hex: :ssl_verify_fun, repo: "hexpm", optional: false]}, {:unicode_util_compat, "~>0.7.0", [hex: :unicode_util_compat, repo: "hexpm", optional: false]}], "hexpm", "de16ff4996556c8548d512f4dbe22dd58a587bf3332e7fd362430a7ef3986b16"},
```
Comment (vanvoljg) [inline mix.lock:13]: Did hackney 1.17.4 fix the issues that we were avoiding with the 1.17.0 pin?

Comment (doughsay) [reply mix.lock:13]: This "shouldn't matter"; it's only used by tzdata for downloading the timezone DB, and the host application can determine the hackney version if there are issues.

Comment (doughsay) [reply mix.lock:13]: To clarify: here's the entire dep tree of cocktail:

```
cocktail
├── credo ~> 1.5 (Hex package)
│   ├── bunt ~> 0.2.0 (Hex package)
│   ├── file_system ~> 0.2.8 (Hex package)
│   └── jason ~> 1.0 (Hex package)
├── dialyxir ~> 1.0 (Hex package)
│   └── erlex >= 0.2.6 (Hex package)
├── ex_doc ~> 0.23 (Hex package)
│   ├── earmark_parser ~> 1.4.0 (Hex package)
│   ├── makeup_elixir ~> 0.14 (Hex package)
│   │   ├── makeup ~> 1.0 (Hex package)
│   │   │   └── nimble_parsec ~> 0.5 or ~> 1.0 (Hex package)
│   │   └── nimble_parsec ~> 1.1 (Hex package)
│   └── makeup_erlang ~> 0.1 (Hex package)
├── excoveralls ~> 0.10 (Hex package)
│   ├── hackney ~> 1.16 (Hex package)
│   │   ├── certifi ~>2.6.1 (Hex package)
│   │   ├── idna ~>6.1.0 (Hex package)
│   │   │   └── unicode_util_compat ~>0.7.0 (Hex package)
│   │   ├── metrics ~>1.0.0 (Hex package)
│   │   ├── mimerl ~>1.1 (Hex package)
│   │   ├── parse_trans 3.3.1 (Hex package)
│   │   ├── ssl_verify_fun ~>1.1.0 (Hex package)
│   │   └── unicode_util_compat ~>0.7.0 (Hex package)
│   └── jason ~> 1.0 (Hex package)
└── timex ~> 3.6 (Hex package)
    ├── combine ~> 0.10 (Hex package)
    ├── gettext ~> 0.10 (Hex package)
    └── tzdata ~> 0.1.8 or ~> 0.5 or ~> 1.0.0 (Hex package)
        └── hackney ~> 1.0 (Hex package)
```

So for prod (ignoring the excoveralls dev-only dep), hackney is pinned to `hackney ~> 1.0`, so the fact we're using 1.17.4 is only relevant to the cocktail CI and local dev.  It won't impact any downstream users.

Comment (vanvoljg) [reply mix.lock:13]: :partying_face:

**Review by vanvoljg — APPROVED** [review]

**Pull request merged**